compiler/rustc_codegen_ssa/src/back/write.rs

```diff
@@ -383,6 +383,12 @@ impl<B: WriteBackendMethods> CodegenContext<B> {
     }
 }
 
+impl<B: CodegenBackend + WriteBackendMethods> CodegenContext<B> {
+    pub fn parallel(&self) -> bool {
+        self.backend.supports_parallel() && !self.opts.unstable_opts.no_parallel_llvm
```

Comment:
the name about `llvm` seems suboptimal (or should only be checked in codegen_llvm)

Reply:
maybe `no_parallel_backend`? 

And `backend` is one of parameters of `start_executing_work` so we can call `supports_parallel` inside it